spec/models/video_spec.rb

```diff
@@ -2,98 +2,56 @@
 
 
 describe Video do
-  it { should belong_to :category }
-  it { should validate_presence_of :title }
-  it { should validate_presence_of :description }
+  it { is_expected.to belong_to :category }
+  it { is_expected.to validate_presence_of :title }
+  it { is_expected.to validate_presence_of :description }
+  it { is_expected.to have_many :reviews }
+
+  describe ".search_by_title" do
+    let!(:videos) do
+      videos = 5.times.map { |i| Fabricate(:video, title: "Lost #{i+1}") }
```

Comment:
For unique titles, you could study [Fabrication docs](http://www.fabricationgem.org/#!sequences) a little bit on sequences. Especially the last one.